manager/logbroker/subscription.go

```diff
@@ -182,6 +183,11 @@ func (s *subscription) match() {
 				continue
 			}
 			for _, task := range tasks {
+				// if we're not following, don't add tasks that aren't ready yet
+				if !s.follow() && task.Status.State < api.TaskStateRunning {
+					s.errors = append(s.errors, errors.Errorf("task %v is not yet running", task.ID))
```

Comment:
I'm not sure, honestly. Probably not? But we do through an error, for example, if a task is not scheduled.